src/blueapi/cli/cli.py

```diff
@@ -231,6 +243,17 @@ def on_event(
     help="Timeout for the plan in seconds. None hangs forever",
     default=None,
 )
+@click.option(
+    "-i",
+    "--instrument-session",
+    type=str,
+    help=textwrap.dedent("""Instrument session associated with running the plan,
+    used to tell blueapi where to store any data and as a secuirty check:
+    the session must be valid and active and you must be a member of it.
+    If you have saved your current session with
+    blueapi controller set-instrument-session, passing this will superseed that."""),
```

Comment:
```suggestion
    help=textwrap.dedent("""
        Instrument session associated with running the plan,
        used to tell blueapi where to store any data and as a security check:
        the session must be valid and active and you must be a member of it.
        If you have saved your current session with
        blueapi controller set-instrument-session, passing this will supersede that."""),
```

dedent does nothing if there's no indentation on the first line (+typos)